caravel/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -17,11 +17,21 @@ import { fDuration } from '../../modules/dates';
 class QueryTable extends React.Component {
   constructor(props) {
     super(props);
+    const uri = window.location.toString();
+    const cleanUri = uri.substring(0, uri.indexOf('#'));
     this.state = {
+      cleanUri,
       showVisualizeModal: false,
       activeQuery: null,
     };
   }
+  getQueryLink(dbId, sql) {
```

Comment:
i think we are using a similar getQueryLink method in a few components. can we generalize and add to a utils file?